src/main.jsx

```diff
@@ -1,10 +1,13 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App'
+import Calculator from './Components/Calculator'
+import Button from './Components/Button'
 import './index.css'
 
 ReactDOM.createRoot(document.getElementById('root')).render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```

Comment:
- [ ] Kindly import the `App` component here and render it. That way you would be implementing the project requirements perfectly. 👍

Reply:
This change has been done well